DEVELOPMENT.MD

```diff
@@ -1,14 +0,0 @@
-## Development Workflow
```

Comment:
Are we deleting this because it's no longer needed or because we don't want to ship it to CRAN?

In the former case, great!

In the latter case, is there another place we can store it? If not should we move it to the Wiki?

Reply:
@jmshapir we're deleting it because we don't want to ship it to CRAN, it's a non-standard file to have in the top directory of an R package so `devtools::check()` was flagging it.

I moved it to the wiki [here](https://github.com/JMSLab/eventstudyr/wiki/Development-Workflow), does this solution work for you?

Reply:
@nateschor that location works for me, but do we need this documentation any longer?

My thoughts are:
* We've implemented most of the design doc so it no longer seems relevant.
* We now have active Github Actions workflow so testing is automatic.
* The overall template for the repository should be clear from the other functions we've written.

In light of these, can we substantially shorten or perhaps even remove the Wiki page?

Reply:
@jmshapir we no longer need this documentation for `eventstudyr`. I agree with all of your bullet points!

I think the documentation is a helpful reference for future package development (I recall @rcalvo12 thinking the same thing). We can remove step 4 since it is hard coded to the `eventstudyr` design document, but the other steps generalize to any package.

Is there a place we could move it for "long term storage"? The [workflow](https://github.com/gslab-econ/lab-manual/wiki/Workflow) or [R](https://github.com/gslab-econ/lab-manual/wiki/R) section of the [gslab lab manual](https://github.com/gslab-econ/lab-manual)?

Reply:
Got it, thanks @nateschor!

I've removed the page from the wiki.

When we wrap up this pull we can include a stable link to the doc in the summary comment. That way we can easily reference it later. If we find another use case for this workflow, we can think about incorporating it into the lab manual.